File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/premium-listing-transaction.ts
import { InvoiceStatusEnum } from 'projects/shared-models-lib/src/lib/enums/invoice-status-enum';
export class PremiumListingTransaction {
  id: number;
  policyId: number;
  rolePlayerId: number;
  invoiceDate: Date;
  invoiceAmount: number;
  paymentDate: Date;
  paymentAmount: number;
  invoiceStatus: InvoiceStatusEnum;
  createdDate: Date;
  statusText: string;
}

File: Service/Billing/RMA.Service.Billing.Contracts/Entities/PremiumListingTransaction.cs
using RMA.Common.Entities;
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Billing.Contracts.Entities
{
    public class PremiumListingTransaction : AuditDetails
    {
        public int RolePlayerId { get; set; } // RolePlayerId
        public int? PolicyId { get; set; } // PolicyId
        public System.DateTime? InvoiceDate { get; set; } // InvoiceDate
        public double? InvoiceAmount { get; set; } // InvoiceAmount
        public System.DateTime? PaymentDate { get; set; } // PaymentDate
        public double PaymentAmount { get; set; } // PaymentAmount
        public InvoiceStatusEnum InvoiceStatus { get; set; } // PaymentStatusId
    }
}